runtimes/spiritnet/src/lib.rs

```diff
@@ -485,7 +497,7 @@ impl pallet_membership::Config<TechnicalMembershipProvider> for Runtime {
 	type WeightInfo = weights::pallet_membership::WeightInfo<Runtime>;
 }
 
-type TipsMembershipProvider = pallet_membership::Instance2;
+type TipsMembershipProvider = pallet_membership::Instance3;
 impl pallet_membership::Config<TipsMembershipProvider> for Runtime {
```

Comment:
And also the callbacks for `MembershipInitialized` and `MembershipChanged` would have to be set to `TipperCommittee` so that changes are reflected in the underlying storage.